<ui version="4.0" >
 <class>MainSettingsPage</class>
 <widget class="QWidget" name="MainSettingsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>347</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy>
    <hsizetype>0</hsizetype>
    <vsizetype>0</vsizetype>
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_4" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>5</hsizetype>
       <vsizetype>0</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string>On program start</string>
     </property>
    </widget>
   </item>
   <item rowspan="2" row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3" >
    <widget class="QCheckBox" name="singleInstanceChk" >
     <property name="toolTip" >
      <string>Use existing instance of application when open files</string>
     </property>
     <property name="text" >
      <string>Open files in existing instance</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <widget class="QCheckBox" name="makeBackupChk" >
     <property name="text" >
      <string>Make backup copy when save file</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" >
    <widget class="QRadioButton" name="showSessionDlgBtn" >
     <property name="text" >
      <string>Show session dialog</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" >
    <widget class="QRadioButton" name="openEmptySessionBtn" >
     <property name="text" >
      <string>Open empty session</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="2" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>188</width>
       <height>111</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" >
    <widget class="QCheckBox" name="autoSaveChk" >
     <property name="text" >
      <string>Auto save every</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2" >
    <widget class="QSpinBox" name="autoSaveIntervalSpin" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="suffix" >
      <string> minute(s)</string>
     </property>
     <property name="prefix" >
      <string/>
     </property>
     <property name="maximum" >
      <number>30</number>
     </property>
     <property name="minimum" >
      <number>1</number>
     </property>
     <property name="value" >
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="1" column="2" >
    <widget class="QRadioButton" name="openLastSessionBtn" >
     <property name="text" >
      <string>Open last saved session</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QCheckBox" name="saveSessionChk" >
     <property name="text" >
      <string>Save session when close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>showSessionDlgBtn</tabstop>
  <tabstop>openLastSessionBtn</tabstop>
  <tabstop>openEmptySessionBtn</tabstop>
  <tabstop>saveSessionChk</tabstop>
  <tabstop>singleInstanceChk</tabstop>
  <tabstop>makeBackupChk</tabstop>
  <tabstop>autoSaveChk</tabstop>
  <tabstop>autoSaveIntervalSpin</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>autoSaveChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>autoSaveIntervalSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>68</x>
     <y>232</y>
    </hint>
    <hint type="destinationlabel" >
     <x>287</x>
     <y>232</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
